src/core/ContractCallStack.ts

```diff
@@ -1,13 +1,14 @@
-import { mapReplacer } from '../utils/utils';
 import { InteractionData } from './modules/impl/HandlerExecutorFactory';
+import { randomUUID } from 'crypto';
```

Comment:
verify if it want break the browser and those dumb web bundlers...it should not - https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility, but you never know...